arch/xtensa/src/esp32/esp32_textheap.c

```diff
@@ -57,16 +51,11 @@ struct mm_heap_s g_textheap;
 
 void up_textheap_init()
 {
-  mm_initialize(&g_textheap, &_stextheap, &_etextheap - &_stextheap);
-
-#if defined(CONFIG_FS_PROCFS) && !defined(CONFIG_FS_PROCFS_EXCLUDE_MEMINFO)
-  static struct procfs_meminfo_entry_s g_textheap_procfs;
-
-  g_textheap_procfs.name = "textheap";
-  g_textheap_procfs.mallinfo = (void *)mm_mallinfo;
-  g_textheap_procfs.user_data = &g_textheap;
-  procfs_register_meminfo(&g_textheap_procfs);
+#ifdef CONFIG_ESP32_RTC_HEAP
+  esp32_rtc_heap_initialize();
```

Comment:
One question should we unify the prefix? esp32_rtc[_]heap<->esp32_iramheap?

Reply:
Which one is better?  I personally prefer the smooshed `rtcheap`, but I was told it wasn't too readable.  Actually the coding standard requires underscores to separate the module and the function but discourages further use, so I think `esp32_rtcheap_initialize` should be the "correct" one.

Reply:
Both are fine for me, the consistency is more important than others.